lib/jazzy/themes/fullwidth/assets/js/lunr.min.js

```diff
@@ -0,0 +1,7 @@
+/**
+ * lunr - http://lunrjs.com - A bit like Solr, but much smaller and not as bright - 0.7.0
```

Comment:
Should we update this to 0.7.2? https://github.com/olivernn/lunr.js/releases/tag/v0.7.2